test/functional/test_framework/test_framework.py

```diff
@@ -1122,8 +1157,17 @@ def activate_dip0024(self, expected_activation_height=None):
     def activate_v19(self, expected_activation_height=None):
         self.activate_by_name('v19', expected_activation_height)
 
-    def activate_v20(self, expected_activation_height=None):
-        self.activate_by_name('v20', expected_activation_height)
+    def activate_v20(self, expected_activation_height=None, test_locked_in_phase=False):
+        if test_locked_in_phase:
```

Comment:
implementation for case of `test_locked_in_phase` is completely different with a normal main-flow of this function; beside that this flag is useful and used only in one functional tets.
I think that's better to just have a separate function inside `feature_llmq_rotation.py` instead complicating this function